forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/services/openbanking/customerinfo/FRCustomerInfoConverter.java

```diff
@@ -31,7 +31,7 @@
     public static CustomerInfo toCustomerInfo(FRCustomerInfo frCustomerInfo){
         CustomerInfo customerInfo = new CustomerInfo();
         customerInfo.setInitials(frCustomerInfo.getInitials());
-        customerInfo.setBirthdate(frCustomerInfo.getBirthdate());
+        customerInfo.setBirthdate(frCustomerInfo.getBirthdate().toLocalDate());
         customerInfo.setEmail(frCustomerInfo.getEmail());
```

Comment:
Convert DateTime to LocalTime from FR Object to OBIE Object to comply with the swagger spec of CustomerInfo object generated.